src/main/java/mods/railcraft/world/level/block/entity/multiblock/MultiblockEntity.java

```diff
@@ -0,0 +1,221 @@
+package mods.railcraft.world.level.block.entity.multiblock;
+
+import java.util.Collection;
+
+import javax.annotation.Nullable;
+
+import net.minecraft.block.BlockState;
+import net.minecraft.nbt.CompoundNBT;
+import net.minecraft.nbt.NBTUtil;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.tileentity.TileEntityType;
+import net.minecraft.util.Direction;
+import net.minecraft.util.math.BlockPos;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class MultiblockEntity<T extends MultiblockEntity<T>> extends TileEntity {
+
+  public static final Logger MULTIBLOCK_LOGGER = LogManager.getLogger("Railcraft/MultiblockEntity");
```

Comment:
Rename this to `logger`, use `LogManager.getLogger()` and make it private. (This applies to the other logger too)

Reply:
currently used for testing